### ISSUE TITLE
Can connect using Ethernet 

### ISSUE THREAD

**Issue by brutu583** [merdok/homebridge-webos-tv#346]
I can connect with WiFi no problem. When I connect my tv via Ethernet (and adjust the ip and MAC address) I get no response in HomeKit.

**Comment by merdok** [comment]
Well i guess that either you are typing the wrong data (ip or mac) or simply your TV is in a different network which cannot be reached by your homebridge device.

**Comment by brutu583** [comment]
Edit. On both WiFi and Ethernet I can control the tv when it’s switched on. When I turn it off, I cannot turn it back on. Also when it’s turned off, it puts all my homebridge devices in ‘no response’. Turning on the tv makes all the devices work again.

**Comment by merdok** [comment]
Then you probably missed to enable a setting on your TV, but this is certainly a network issue on your side or a setting on the TV. BTW, you did not state what TV do you have, can you share that?

**Comment by brutu583** [comment]
TV is LG UM7000PLA

After all night trying everything from the tv setting, to my network, to even restoring my phone after screwing up homebridge, it was the Mobile TV On setting that needed to be enabled. 

Funny thing is I enabled it before and it didn’t work. This morning after reinstalling the plugin for the millionth time, I noticed it was off. Turned it on and now everything works. 

Oh well, case closed!
